public/locales/pl/translation.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Why is this whole file added? 🤔  It's already on main.

Reply:
Is there a correct base branch picked for this feature branch? Not in the PR, as I see it is main here, but actually the branch you were starting locally at first?

Reply:
Once when i was  pulling main branch I used merge not rebase maybe because of that my PR looks like that

Reply:
Hm, interesting, maybe  :D

Reply:
Once more time I rebase and push it with force